src/cmr/concepts/aclConcept.js

```diff
@@ -0,0 +1,344 @@
+import snakeCaseKeys from 'snakecase-keys'
+
+import { pick, snakeCase } from 'lodash'
+
+import { CONCEPT_TYPES } from '../../constants'
+
+import { aclQuery } from '../../utils/aclQuery'
+import { mergeParams } from '../../utils/mergeParams'
+import { parseError } from '../../utils/parseError'
+import { downcaseKeys } from '../../utils/downcaseKeys'
+
+export default class AclConcept {
+  /**
+   * Instantiates a Concept object
```

Comment:
to general, how is this one different from Acl? Where does this come from? Would sample JSON help for both cases?

Reply:
The Concept.js is generally for Collection, Variable, and Tool; therefore; Deep and I decided to create this component name as AclConcept is just for ACL.  There are some re-used functionalities from the Concept.js